openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiCL.java

```diff
@@ -91,8 +99,17 @@ public void run(String jarName)
 				{
 					cntr++;
 					Dummy myDummy = (Dummy)myC.newInstance();
-					if(cntr % 1000 == 0)
+					if(cntr % 1000 == 0) {
 						logMessage("Loaded " + cntr + " classes...");
+						
+						// This flag has been added to terminate the load test within 10 minutes 
+						// keeping in mind that the test launches 5 processes to simultaneously run this load and 
+						// the overall time it takes is especially slow on certain platforms (e.g. Windows). 		
+						long durationInMinutes = (System.currentTimeMillis() - startTime) / (1000 * 60);  
+						if (durationInMinutes > 10 ) {
+							loadDone = true; 
```

Comment:
I think this should output a message indicating the test duration expired.

Reply:
Added message.